src/jobs/JobsForUser.js

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState, useContext } from "react";
+import SearchForm from "../shared/SearchForm";
+import JoblyApi from "../api";
+import JobCardList from "../jobs/JobCardList";
+import LoadingSpinner from "../LoadingSpinner";
+import userContext from "../userContext";
+/** JobList
+ * 
+ * Props:
+ *  - applyForJob - fn to call when applying for job called in JobCard
+ * 
+ * State:
+ * - jobs: Array of job objects
+ *    [{ id, title, salary, equity }, ...]
+ * - queryTerm: string to filter the job results
+ * - isLoading: boolean
+ *  
+ * 
+ */
+
+function JobListForUser({applyForJob}) {
```

Comment:
Lets make a general component for this where the 'job retrieval' function can be passed down as a prop